src/components/date-input/README.md

```diff
@@ -29,32 +29,37 @@ More information about when to use date-input can be found on [GOV.UK Design Sys
 
       </legend>
 
-      <div class="govuk-c-date-input" id="dob">
+      <div class="govuk-o-form-group govuk-o-form-group--error">
+        <div class="govuk-c-date-input" id="dob">
 
-        <div class="govuk-c-date-input__item govuk-c-date-input__item--day">
-          <label class="govuk-c-label govuk-c-date-input__label" for="dob-day">
+          <div class="govuk-c-date-input__item govuk-c-date-input__item--day">
+            <div class="govuk-o-form-group"><label class="govuk-c-label govuk-c-date-input__label" for="dob-day">
```

Comment:
Is this html rendering as expected?

Reply:
I've just tried to improve the indentation of the readme. I think adding a wrapper to the component has thrown it out.

Reply:
As @nickcolley has just pointed out there are still some text indentation issues in the README. I don't think fiddling with the spacing and trimming and what not is a good use of my time 😞   if we're looking to implement Prettify. I think if we decide not to use Prettify, we should look at this issue again. Contributors will keep running into it as well.